suites/suite_rayleigh_friction.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would move this file from `suites` to `test/test_suites`, as it will likely only be used for testing.

Reply:
I didn't know about test/test_suites. Maybe it should be added to the ccpp-ization instructions as an option?

Reply:
done!